docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -164,3 +164,111 @@ The Divider's `light` prop was deprecated, Use `sx={{ opacity : "0.6" }}` (or an
 +  sx={{ opacity : "0.6" }}
  />
 ```
+
+## PaginationItem
+
+Use the [codemod](https://github.com/mui/material-ui/tree/HEAD/packages/mui-codemod#pagination-item-classes) below to migrate the code as described in the following sections:
+
+```bash
+npx @mui/codemod@latest deprecations/pagination-item-classes <path>
+```
+
+### .MuiPaginationItem-textPrimary
```

Comment:
We should summarize all `PaginationItem`'s composed classes deprecations into a single section. Otherwise, this file will get huge 😅. We can do:

```md
### Composed CSS classes

The following CSS classes were deprecated:

- `.MuiPaginationItem-textPrimary` deprecated in favor of `.MuiPaginationItem-text.MuiPaginationItem-colorPrimary`
- `.MuiPaginationItem-outlinedPrimary` deprecated in favor of `.MuiPaginationItem-outlined.MuiPaginationItem-colorPrimary`
- ...

Here's an example on how to migrate:

(Example diffs for `.MuiPaginationItem-textPrimary`)
```

Reply:
what do you think of current format https://deploy-preview-41145--material-ui.netlify.app/material-ui/migration/migrating-from-deprecated-apis/#paginationitem

Reply:
I would remove the list to be more concise:

```md
### Composed CSS classes

The CSS classes that composed the `variant` and `color` prop values were removed. Here's how to migrate:

(... diffs)
```